### PR TITLE
data:ravi0900

### PR DESCRIPTION
**Pull request by ravi0900**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3404"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Your filename must match your GitHub username, make it `ravi0900.json`

File: data/ravirajkushwaha.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Raviraj Kushwaha",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I'm kind of Noob and a tech nerd, Watching so many tech updates and learning little bit as well",
+  "avatar": "https://raw.githubusercontent.com/ravi0900/AboutMe/main/images/image.png",
```
Comment (Aadarsh805) [inline data/ravirajkushwaha.json]: use this link for avatar `https://github.com/ravi0900.png`

File: data/ravirajkushwaha.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "Links": [
```
Comment (Aadarsh805) [inline data/ravirajkushwaha.json]: this `L` in `Links` should be `l` (small L)

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Comment by Aadarsh805** [comment]
@ravi0900 now your file is out of `data` directory, for this change your name to `data/ravi0900.json`

**Comment by ravi0900** [comment]
> @ravi0900 now your file is out of `data` directory, for this change your name to `data/ravi0900.json`

now everything fixed ? right ?

**Review by Aadarsh805 — APPROVED** [review]
Yep all good 👍

**Review by ChinmayMhatre — APPROVED** [review]

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good!

**Pull request merged**